### ISSUE TITLE
Time running out

### ISSUE THREAD

**Issue by pedro-w** [grrk-bzzt/kq-fork#133]
We started on this period of activity after Z suggested we could try and get to 1.0 by September, which is now less than a month away. Is this still feasible - can we prioritize just what's needed to get the game done without adding new features to/reworking the C++ code? Things like changing the character sprite size are not going to be possible I don't think.
As far as the plot goes, are we happy it all flows through to the end? I know the goblin thing is not really finished. Also I suspect there are a few bits of dialog which are a bit sketchy? What do you think @OnlineCop  @z9484 ?

**Comment by OnlineCop** [comment]
The engine rework that I think is still absolutely necessary was related to #124 (eResistance) since the data being pulled in, and how it's actually consumed by the engine, don't always match. Some of the elemental effects are inconsistent:
- "Rod of Fire" has `elem == eResistance::R_FIRE` (0-based)
- "Rune of Fire" has `elem == eResistance::R_BLACK` (1-based)

It requires engine rework because SOME of the code paths expect the values to be 1-based but it passes that same 1-based value around (like `csmem` and `ctmem`), such as fighter "resistance damage calculation", where the 1-based value is expected to have been converted to 0-based (which it hasn't).

- I think the best thing to do here is to change the engine to expect 0-based, and update the data in the allstat.mon and resabil.mon data to use the MAX/TOTAL values to mean "no effect".

**Comment by OnlineCop** [comment]
I believe #31 should still be addressed: the battles between Manor and Ekla (town1) are tolerable when there is 1 enemy against 1 party member, but until I'm nearly Level 4, any battle with 2 enemies will wipe me out.

All enemies in this zone should have their attack damage nerfed.
- If we want to keep the 2 enemies, they should only be able to do 1-3 HP worth of damage apiece, else a preemptive strike will almost always guarantee the player will be killed.
- If we want to reduce the total number of enemies to 1, they should be able to do 4-6 HP worth of damage since the player should be able to go a few rounds before requiring healing.

In most JRPGs I've played, enemies in the starting zone do only 1-2 damage apiece, which is survivable even when the player's starting MHP is low (10-20).

**Comment by z9484** [comment]
Yes I think that would work to change the forests around the manor to have only 1 enemy.

**Comment by pedro-w** [comment]
OK, let's change that as z9484 suggested

**Comment by pedro-w** [comment]
I'm not going to change it just yet because @OnlineCop  is in the midst of #124 which I think will completely alter the format of the battle table?
